### PR TITLE
Fixing hack handling whether the app is deployed or not.

### PR DESCRIPTION
**Pull request by meg-huggingface**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -11,17 +11,26 @@
...
 
 import streamlit as st
 
 from data_measurements import dataset_statistics, dataset_utils
 from data_measurements import streamlit_utils as st_utils
 
+"""
+Examples:
+# When not in deployment mode
+streamlit run app.py -- --live=False
```
Comment (TristanThrush) [inline app.py:28]: is there an extra `--` here by mistake?

**Review by TristanThrush — APPROVED** [review]
LGTM, except the thing that I ask about below!

File: app.py (hunk 1 above)
Comment (TristanThrush) [reply app.py:28]: Also, with argparse, you can do this to store booleans if you dont want to add the `=False` in the command line. But either way seems ok to me!

```
parser.add_argument('--foobar', action='store_true')
parser.add_argument('--foonley', action='store_false')
```

Comment (meg-huggingface) [reply app.py:28]: Nope!

And thanks! I always use store_true, forgot about store_false !!!

**Pull request merged**